app/main.js

```diff
@@ -60,11 +60,15 @@ function displaysY (height = 600) {
 }
 
 function createTrayIcon () {
```

Comment:
Haven't had a chance to test it on Linux, but that should restore the original behavior.